src/CommunityToolkit.Maui/AppBuilderExtensions.shared.cs

```diff
@@ -17,9 +17,12 @@ public static class AppBuilderExtensions
 	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui"/></returns>
 	public static MauiAppBuilder UseMauiCommunityToolkit(this MauiAppBuilder builder, Action<Options>? options = default)
 	{
+		builder.UseMauiCommunityToolkitCore(options is not null
+											? new Action<Core.Options>(coreOptions => options((Options)coreOptions))
+											: null);
+
```

Comment:
Hey Vlad! I found a way to cast `Action<Options>` to `Action<Core.Options>`: https://stackoverflow.com/a/3444404/5953643

Reply:
what should we do with lines 24-25? it will be different options? do we want both?

Reply:
Whoops! I just realized that since `Options : Core.Options`, we only need to run `options?.Invoke(options)` once.

In other words, we can safely call `builder.UseMauiCommunityToolkitCore(null);` in `UseMauiCommunityToolkit()` because we know for a fact that the options in `Core.Options` have been set.

Reply:
I fixed it in this commit: https://github.com/CommunityToolkit/Maui/pull/616/commits/2b45efc8ca1c3c50663bb4bb6dd0a083bc10afe4